packages/release-manifest/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@backstage/release-manifest",
+  "description": "Helper library for receiving release manifests",
+  "version": "0.0.0",
+  "private": false,
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "packages/release-manifest"
+  },
+  "keywords": [
+    "backstage"
+  ],
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "backstage-cli build --outputs cjs,types",
```

Comment:
I realized we kinda might as well make this isomorphic, so that you consume this from frontend plugins too. Perhaps a followup though